src/components/CategoryFilter.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+class CategoryFilter extends Component {
+  handleFilterChange = category => {
+    const { bookFilter } = this.props;
+    bookFilter(category);
+  }
+
+  render() {
+    const categories = [
```

Comment:
- [ ] All is not a valid book category. 

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone4/AllReturnsAllCategories.png)

Reply:
Thank you for your feedback.

The last part of the statement says **...although it should be a filter option**, and that is exactly what we did. We know that our `All` should not be saved in the Redux store and we never did. I urge the next reviewer to please check out the `filter` reducer and see how we ensured that the `All` option is not stored in our database.

Reply:
### [Optional]
According to the instruction, our CategoryFilter should be a presentational component;
![image](https://user-images.githubusercontent.com/31897434/97775686-04224880-1b63-11eb-9ef1-fa96a85894e9.png)
therefore; our CategoryFilter should not be connected to the redux store.